packages/animations/browser/src/dsl/animation_ast_builder.ts

```diff
@@ -78,7 +78,7 @@ import {AnimationDslVisitor} from './animation_dsl_visitor';
 import {parseTransitionExpr} from './animation_transition_expr';
 
 const SELF_TOKEN = ':self';
-const SELF_TOKEN_REGEX = new RegExp(`s*${SELF_TOKEN}s*,?`, 'g');
+const SELF_TOKEN_REGEX = /* @__PURE__ */ new RegExp(`s*${SELF_TOKEN}s*,?`, 'g');
```

Comment:
Why not just inline the string constant for both minification and dead code elimination?
```suggestion
const SELF_TOKEN_REGEX = /`s*:selfs*,?/g;
```

Not sure I see the value in the current version or the new alternative, seems like a waste of bytes to me